### ISSUE TITLE
View of multiple result sets is horrible

### ISSUE THREAD

**Issue by SeriousM** [Microsoft/azuredatastudio#4820]
Issue Type: <b>Bug</b>

When I select multiple result sets the view of them is just very bad visualized.

The width of the grid is just confusing and there is no visual distinction to help the eye where to look at.

![image](https://user-images.githubusercontent.com/462013/55481533-e3aaa900-5622-11e9-94a0-6e6bbd5f271d.png)

And scrolling, oh my... please just fix that.

![image](https://user-images.githubusercontent.com/462013/55481638-1ce31900-5623-11e9-875f-d9c00f0cf825.png)

Azure Data Studio version: azuredatastudio 1.5.2 (f74080c96310354bc92211f1826c0fdce78ca2f6, 2019-03-22T06:25:26.627Z)
OS version: Windows_NT x64 10.0.17763

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 x 2808)|
|GPU Status|2d_canvas: enabled<br>checker_imaging: disabled_off<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|31.86GB (9.63GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (4)</summary>

Extension|Author (truncated)|Version
---|---|---
agent|Mic|0.37.0
profiler|Mic|0.7.0
server-report|Mic|0.1.5
whoisactive|Mic|0.1.1

</details>

**Comment by kburtram** [comment]
@SeriousM regarding the second screenshot, it looks like there are multiple containers with scrollable content (3+ resultgrids and the container area for the resultgrids).  How were you expecting the scrollbars to be arranged to support scrolling these various containers?

Not saying it's the best UX, but that screenshot looks fairly similar to how we handle this layout in SSMS as far as I can tell (screenshot below for reference).  Thanks for the feedback!

![image](https://user-images.githubusercontent.com/599935/55514233-654b0880-561c-11e9-9ec8-2d84f4db906d.png)

**Comment by SeriousM** [comment]
@kburtram You're right and I wasn't happy with the solution from SSMS either, but influencing the ui design of SSMS was rather hard these days.
Anyway, I guess the solution should be
1. add a visual border as in SSMS so you know which scrollbar belongs to which list - think of different grid types with unequal records in each
2. have the scroll focus on the outer container. when clicking into one of the nested containers, this one should scroll.
3. the grid maximize button should be invoked on double-click on a nested container. 
![image](https://user-images.githubusercontent.com/462013/55539267-8bc27f80-56c0-11e9-8b06-f0fb9fd96776.png)

Whats your thoughts?

**Comment by datauduong** [comment]
I agree that viewing multiple result set as single view (scroll-able) is not user friendly, I would prefer each result in separate tab.
